Remotion/Web/Core/PlainTextString.cs

```diff
@@ -0,0 +1,220 @@
+    /// <returns>The newly created <see cref="PlainTextString"/> containing the <paramref name="text"/>.</returns>
+    public static PlainTextString CreateFromText ([CanBeNull] string? text)
+    {
+      return new PlainTextString (text);
+    }
+
+    /// <summary>
+    /// Implements the equality operator for <see cref="PlainTextString"/>s. The operator is implemented the same way as the <see cref="Equals(PlainTextString)"/> method.
+    /// </summary>
+    /// <param name="left">The first value to be compared for equality.</param>
+    /// <param name="right">The second value to be compared for equality.</param>
+    /// <returns>
+    ///   <see langword="true" /> if both <paramref name="left"/> and <paramref name="right"/> have the same string value. Otherwise, <see langword="false" />.
+    /// </returns>
+    public static bool operator == (PlainTextString left, PlainTextString right)
+    {
+      return left.Equals (right);
+    }
+
+    /// <summary>
+    /// Implements the inequality operator for <see cref="PlainTextString"/>s. The operator is implemented the same way as the <see cref="Equals(PlainTextString)"/> method.
+    /// </summary>
+    /// <param name="left">The first value to be compared for inequality.</param>
+    /// <param name="right">The second value to be compared for inequality.</param>
+    /// <returns>
+    ///   <see langword="true" /> if both <paramref name="left"/> and <paramref name="right"/> do not have the same string value. Otherwise, <see langword="false" />.
+    /// </returns>
+    public static bool operator != (PlainTextString left, PlainTextString right)
+    {
+      return !left.Equals (right);
+    }
+
+    /// <summary>
+    /// Implements the implicit conversion from <see cref="PlainTextString"/> to <see cref="WebString"/>.
+    /// </summary>
+    /// <param name="value">The <see cref="PlainTextString"/> to be converted.</param>
+    /// <returns>A <see cref="WebString"/> of type <see cref="WebStringType"/>.<see cref="WebStringType.PlainText"/>.</returns>
+    public static implicit operator WebString (PlainTextString value)
+    {
+      return WebString.CreateFromText (value._value);
+    }
+
+    /// <summary>
+    /// Implements the explicit conversion from <see cref="string"/> to <see cref="PlainTextString"/>.
+    /// </summary>
+    /// <param name="value">The <see cref="string"/> to be converted.</param>
+    /// <returns>A <see cref="PlainTextString"/> with the specified <paramref name="value"/>.</returns>
+    public static explicit operator PlainTextString (string value)
+    {
+      return new PlainTextString (value);
+    }
+
+    [CanBeNull]
+    private readonly string? _value;
+
+    private PlainTextString ([CanBeNull] string? value)
+    {
+      _value = value;
+    }
+
+    /// <summary>
+    /// Tests whether the <see cref="PlainTextString"/>s value is <see langword="null" /> or an empty string.
```

Comment:
This is a good example where using "Sytem.String" instead "string" would be overkill / noise, even though strictly speaking, more correct.